### ISSUE TITLE
Allow Linux install as user (not root)

### ISSUE THREAD

**Issue by matthewdarwin** [Boid-John/boidcmd#4]
I installed boinc-client ahead of time, as suggested in the readme.

Then I went to install this client into my local directory, and it still prompts me for superuser access.  Why? I don't want to run anything as 'root'.

```
npm install boidcmd
npm WARN npm npm does not support Node.js v10.15.1
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 4, 6, 7, 8, 9.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN saveError ENOENT: no such file or directory, open '/home/matthew/package.json'
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN enoent ENOENT: no such file or directory, open '/home/matthew/package.json'
npm WARN matthew No description
npm WARN matthew No repository field.
npm WARN matthew No README data
npm WARN matthew No license field.

+ boidcmd@1.0.5
added 36 packages from 44 contributors in 1.348s

$ ./node_modules/boidcmd/bin/boidcmd setup
Boid Account Email:xxxxxxxxxxxxxxxx
Boid Account Password: *********[sudo] password for matthew:
error { Error: Command failed:
            sudo apt install boinc-client -y
            boinccmd --project_attach http://www.worldcommunitygrid.org/ 1061556_a0c611b081f8692b7ef0c11d39e6105c

sudo: no password was provided
can't connect to local host

    at ChildProcess.exithandler (child_process.js:294:12)
    at ChildProcess.emit (events.js:189:13)
    at maybeClose (internal/child_process.js:970:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
  killed: false,
  code: 1,
  signal: null,
  cmd:
   '\n            sudo apt install boinc-client -y\n            boinccmd --project_attach http://www.worldcommunitygrid.org/ 1061556_a0c611b081f8692b7ef0c11d39e6105c\n        ' }

```

**Comment by matthewdarwin** [comment]
I'm comfortable to set things up manually if you tell me what needs to be done.

I can run the tool
```
boinc --allow_remote_gui_rpc --dir /home/matthew/boinc --attach_project http://www.worldcommunitygrid.org/ 1061556_a0c611b081f8692b7ef0c11d39e6105c
```

**Comment by ghobson2013** [comment]
There seems to be some confusion in this issue, 

Let me clarify a few things:
   * boidcmd is a wrapper application that simply calls boinccmd
   * boinccmd is installed with the boinc-client , which you pre-installed, but we can't assume it's always pre-installed so we go ahead and try to install it. As a non-root user you have to use sudo for this to work. This is the only time sudo is needed. After that all commands simply call boinccmd. 
   * boinc-client is running as the user "boinc"  , if you do ps -aux you will see the worldgridcommunity processes owned by boinc.

So to conclude at no time do we run anything as root, except on first install when we need you to install the boinc-client application , that's it.

I hope a clarified this issue.

**Comment by boid-com** [comment]
This issue might be resolved with pr #6. Feel free to open this issue again if you find issues with the latest version.
